pkg/process/runner/runner.go

```diff
@@ -501,22 +505,27 @@ func readResponseStatuses(checkName string, responses <-chan defaultforwarder.Re
 		r, err := model.DecodeMessage(response.Body)
 		if err != nil {
 			log.Errorf("[%s] Could not decode response body: %s", checkName, err)
+			submissionErrors++
 			continue
 		}
 
 		switch r.Header.Type {
 		case model.TypeResCollector:
```

Comment:
Tangent: Is this case even still used? 🤔 

(Not necessarily for @gengnamstyle to respond, if nobody knows the answer I can look into it)

Reply:
This is still the type used by process-intake when sending responses so I think so?
https://github.com/DataDog/dd-go/blob/prod/process/apps/process-intake/api.go#L404

Reply:
Ohhh, I thought `ResCollector` meant "resource collector", my bad